pkg/verifier/gcp/gcp_verifier.go

```diff
@@ -92,6 +92,11 @@ func (g *GcpVerifier) createComputeServiceInstance(input createComputeServiceIns
 			{
 				Name:       input.networkName,
 				Subnetwork: input.vpcSubnetID,
+				AccessConfigs: []*computev1.AccessConfig{
+					{
+						Name: "External NAT",
+					},
+				},
```

Comment:
Not a blocker, but we should discuss this further (or at least put a TODO comment). It looks like this assigns a public IP to the instance, which was something we were asked to stop doing on the AWS side of things long ago. Please correct me if I'm misunderstanding GCP access configs though

Reply:
Not 100% on this but https://pkg.go.dev/google.golang.org/api@v0.114.0/compute/v1#AccessConfig says 
```
	// AccessConfigs: An array of configurations for this interface.
	// Currently, only one access config, ONE_TO_ONE_NAT, is supported. If
	// there are no accessConfigs specified, then this instance will have no
	// external internet access.
```
which from what I have experienced would block egress as well. However, that is not to say there are no workarounds.

Reply:
Ah thanks for clarifying! Sounds like this is spot-on then. Could you add a comment along the lines of "// Must specify an AccessConfig in order to enable VM internet access"?